stripe/src/test/java/com/stripe/android/view/CardInputViewTest.java

```diff
@@ -79,17 +81,17 @@ public void onDeleteFromExpiryDate_whenEmpty_shiftsFocusToCardNumber() {
     public void onUpdateIcon_forCommonLengthBrand_callsSetImageResourceAndSetsLengthOnCvc() {
         // This should set the brand to Visa. Note that more extensive brand checking occurs
         // in CardNumberEditTextTest.
-        Drawable old = mIconView.getDrawable();
+        Bitmap oldBitmap  = ((BitmapDrawable) mIconView.getDrawable()).getBitmap();
```

Comment:
When switching to a tint model, checking on the drawable didn't register that there had been a change.